UPGRADING.md

```diff
@@ -1,6 +1,18 @@
 Upgrading
 =========
 
+## 6.x to 7.x
+
+### The new Exception interceptor
+
+When building using Unity 2019+, the Bugsnag SDK now uses a new method to intercept uncaught C# exceptions. This allows us access to the original exception object, meaning more accurate exception data and full support for nested/inner exceptions.
+
+This means that uncaught C# exceptions previously reported will not group automatically in the Bugsnag dashboard with exceptions reported by the new notifier. 
```

Comment:
Should we add a bit more info here as it doesn't really say _why_ it won't group.

e.g.

> Due to differences in the way types were previously exposed in Unity's logging mechanism and the type information available when the exception is intercepted, errors will be reported subtly differently once your app is upgraded to this version.

Reply:
Updated my suggestion.